apps/astarte_realm_management/lib/astarte_realm_management/queries.ex

```diff
@@ -1783,24 +1783,10 @@ defmodule Astarte.RealmManagement.Queries do
   end
 
   def retrieve_devices_to_delete!(realm_name) do
-    Xandra.Cluster.run(:xandra_device_deletion, &do_retrieve_devices_to_delete!(&1, realm_name))
-  end
-
-  defp do_retrieve_devices_to_delete!(conn, realm_name) do
-    # TODO: validate realm name
-    keyspace_name =
-      CQLUtils.realm_name_to_keyspace_name(realm_name, Config.astarte_instance_id!())
-
-    statement = """
-    SELECT *
-    FROM #{keyspace_name}.deletion_in_progress
-    """
+    keyspace = Realm.keyspace_name(realm_name)
 
-    Xandra.execute!(conn, statement, %{},
-      consistency: :local_quorum,
-      uuid_format: :binary
-    )
-    |> Enum.to_list()
+    from("deletion_in_progress")
```

Comment:
If we specify the module of an Ecto schema instead of the table name directly, we might have an easier life in the future should we decide to change the underlying table.